cmd/create/service/cmd.go

```diff
@@ -122,7 +137,10 @@ func run(cmd *cobra.Command, argv []string) {
 	awsClient := aws.GetAWSClientForUserRegion(reporter, logger)
 
 	// Get AWS region
-	args.AwsRegion, err = aws.GetRegion(arguments.GetRegion())
+	args.AwsRegion = region.Region()
+	if args.AwsRegion == "" {
+		args.AwsRegion, err = aws.GetRegion(arguments.GetRegion())
+	}
```

Comment:
This change is redundant, `arguments.GetRegion()` already call `region.Region()` internally.